README.md

```diff
@@ -374,13 +376,43 @@ let app =
         ]
 ```
 
+### subRoute
+
+`subRoute` checks the current path begins with the `path` string, all `route*` and `subRoute*` handlers within the `handler` parameter will now be rooted to, and relative with respect to this path.
+
```

Comment:
Yes you are right. Maybe just remove this bit then, like this:

`subRoute` checks if the current path begins with the given subPath and will invoke the passed in `handler` if it was a match. The given `handler` (and any nested handlers within it) should omit the already applied subPath for subsequent route evaluations